src/overlays/gamestates/ovl_select/z_select.c

```diff
@@ -56,8 +56,7 @@ void MapSelect_Init(GameState* thisx) {
     // turning the sfx volume back on
     SEQCMD_SET_SEQPLAYER_VOLUME(SEQ_PLAYER_BGM_MAIN, 0, 10);
 
-#if BOOT_TO_MAP_SELECT
-    gSaveContext.fileNum = 0xFF;
+#if BOOT_TO_DEBUG_OPENING
```

Comment:
what if I wanna use boot to map select instead of the boot menu? I'm a bit confused by this